File: backend/src/main/java/com/project/linkedin/Proffessional/Proffessional.java
package com.project.linkedin.Proffessional;

import lombok.Data;

import javax.persistence.*;


@Data
@Entity
public class Proffessional {


    private @Id @GeneratedValue(strategy = GenerationType.SEQUENCE) Long idProffessional;
    private Long id_user;
    private Long id_information_page;


    public Proffessional() {
    }

    public Proffessional(Long User_id, Long InformationPage_id) {


        this.id_user = User_id;
        this.id_information_page = InformationPage_id;

    }

}

File: frontend/src/app/Proffessional/proffessional.ts
export class proffessional{
    id_proffessional: Number;
    id_user: Number;
    id_information_page: Number;

    constructor(id_user: Number, id_information_page: Number){
        this.id_proffessional = 0;
        this.id_user = id_user;
        this.id_information_page = id_information_page;
    }

    set(id_proffessional: Number, id_user: Number, id_information_page: Number){
        this.id_proffessional = id_proffessional;
        this.id_user = id_user;
        this.id_information_page = id_information_page;
    }
}
